### PR TITLE
Rename console to busola and pamela to backend

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Rename console to busola and pamela to backend

**Related issue(s)**

### CODE REVIEW

File: components/react/src/components/Search/Readme.md [hunk 1]
```diff
@@ -6,7 +8,7 @@
   <h5>Search with placeholder</h5>
   <Search placeholder="Search"/>
 
-  <h5>Search with `onChange` function (see Developer Console for output)</h5>
+  <h5>Search with `onChange` function (see Developer Busolafor output)</h5>
```
Comment (Wawrzyn321) [inline components/react/src/components/Search/Readme.md:11]:
```suggestion
  <h5>Search with `onChange` function (see Developer Busola for output)</h5>
```

File: backend/index.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ initializeApp(app, kubeconfig)
     }
 
     server.listen(port, address, () => {
-      console.log(`👙 PAMELA 👄 server started @ ${port}!`);
+      console.log(`👙 Busola backend 👄 server started @ ${port}!`);
```
Comment (Wawrzyn321) [inline backend/index.js:41]:
```suggestion
      console.log(`Busola backend server started @ ${port}!`);
```
:(

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
Would you kindly change also `fetchConsoleInitData`?

File: backend/index.js (hunk 2 above)
Comment (dariadomagala) [reply backend/index.js:41]: 😢 are you sure?

**Review by Wawrzyn321 — APPROVED** [review]

**Review by valentinvieriu — APPROVED** [review]

**Pull request merged**